src/cnv/ugbio_cnv/combine_cnmops_cnvpytor_cnv_calls.py

```diff
@@ -349,21 +363,21 @@ def combine_cnv_vcfs(
     Combine VCF files from cn.mops and CNVpytor into a single sorted and indexed VCF.
```

Comment:
```suggestion
    Concatenates VCF files from cn.mops and CNVpytor into a single sorted and indexed VCF.
```
Just a suggestion: 
As far as I understand, this function outputs records from multiple VCF files into a single, sorted VCF. The term “combine” suggests some additional, non-trivial processing beyond simple aggregation, whereas “concat” more clearly conveys that the records are merely appended and printed as-is, without further manipulation.